File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.cs
using System.Linq;
using System.Collections.Generic;

public class Solution {
    public int[] solution(int[] numbers) {
        int[] answer = new int[] {};
        answer = numbers;
        List<int> list = new List<int>();
        
        for (int i = 0; i < answer.Length; i++){
            for (int j = 0; j < answer.Length; j++){
                if (i == j) continue;
                list.Add(answer[i] + answer[j]);
            }
        }
        
        list = list.Distinct().ToArray().ToList();
        list.Sort();
        answer = list.ToArray();
        
        return answer;
    }
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.cpp
#include <string>
#include <vector>
#include <set>
using namespace std;
vector<int> solution(vector<int> numbers) {
	

	set<int> result;
	for (int i = 0; i < numbers.size(); i++)
	{
		for (int j = 0; j < numbers.size(); j++)
		{
			if (i == j) continue;

			result.insert(numbers[i] + numbers[j]);

		}
	}

	vector<int> answer;

	for (auto it : result)
	{
		answer.push_back(it);
	}

	return answer;

}
